scenarios/multi_object_tracking/01_training_introduction.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<ul><li>[minor] "greatly enhance its performance." might be a bit confusing, since without training the accuracy would be 0 of course.</li><li>[minor] When referring to code, put inside brackets such as this: <code>unzip()</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>